index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title></title>
```

Comment:
I would suggest you add title to the page since It displays the title for the page in search engine results.